cmds/more/more.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// More pages through files without any terminal trickery.
+//
+// Synopsis:
+//     more [OPTIONS] FILE
+//
+// Options:
+//     --lines NUMBER: screen size in number of lines
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+
+	flag "github.com/spf13/pflag"
+)
+
+var lines = flag.Int("lines", 40, "screen size in number of lines")
+
+func main() {
+	flag.Parse()
+	if flag.NArg() != 1 {
+		log.Fatal("more can only take one file")
+	}
+	if *lines <= 0 {
```

Comment:
typically, you'd get lines by using our termios package.

        tty, err := termios.New()
        if err != nil {
                return err
        }

        if p.WS, err = tty.GetWinSize(); err != nil {
                return err
        }

but that's maybe more work than it's worth ...

Reply:
This is what less does (albeit with termbox). Since we already have less, the aim is for more to differentiate itself by being very very simple.